src/NUnitTestAdapterTests/AdapterSettingsTests.cs

```diff
@@ -171,5 +173,22 @@ public void RandomSeedSetting()
             _settings.Load("<RunSettings><NUnit><RandomSeed>12345</RandomSeed></NUnit></RunSettings>");
             Assert.That(_settings.RandomSeed, Is.EqualTo(12345));
         }
+
+        [Test]
+        public void SynchronousEventsSetting()
+        {
+            _settings.Load("<RunSettings><NUnit><SynchronousEvents>True</SynchronousEvents></NUnit></RunSettings>");
+            Assert.True(_settings.SynchronousEvents);
+        }
+
+        [Test]
+        public void DomainUsageSetting()
```

Comment:
Minor point, but this would probably be better as two tests created using [TestCase(...)] attributes.